backends/bmv2/midend.cpp

```diff
@@ -122,6 +123,11 @@ MidEnd::MidEnd(BMV2Options& options) {
         new P4::CompileTimeOperations(),
         new P4::TableHit(&refMap, &typeMap),
         new P4::RemoveLeftSlices(&refMap, &typeMap),
+
+        // p4c-bm removed unused action parameters. To produce a compatible
+        // control plane API, we remove them as well for P4-14 programs.
+        isv1 ? new P4::RemoveUnusedActionParameters(&refMap) : nullptr,
```

Comment:
just checking, doesn't that mean this issue is going to come up again: https://github.com/p4lang/p4c/issues/497